packages/debug/src/browser/debug-session-manager.ts

```diff
@@ -214,7 +214,7 @@ export class DebugSessionManager {
                     }
                 }
 
-                const sessionId = await this.debug.createDebugSession(resolved.configuration);
+                const sessionId = await this.debug.createDebugSession(resolved.configuration, options.workspaceFolderUri);
```

Comment:
It does not appear that the implementation of the `createDebugSession` method on the `DebugServiceImpl` has been updated to handle the `workspaceFolderUri`. Is that intentional?